.github/workflows/check.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         env:
           ORG_GRADLE_PROJECT_signingKey: ${{ secrets.SIGNING_KEY }}
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.SIGNING_PASSWORD }}
-        run: ./gradlew check
+        run: ./gradlew check --info
```

Comment:
please remove the info logging again

Reply:
Instead we maybe should attach the junit result to the action.
But for this small project I think it is fine to run a failing test locally to get more information.